src/axolotl/train.py

```diff
@@ -12,6 +12,7 @@
 from accelerate.logging import get_logger
 from datasets import Dataset
 from optimum.bettertransformer import BetterTransformer
+from pkg_resources import get_distribution  # type: ignore
```

Comment:
@winglian `mypy` hates this specific import, and wants a special package to be installed.  I wrestled with it for a while, but made a judgement call that ignoring this with `mypy` is ok